include/zephyr/bluetooth/iso.h

```diff
@@ -217,6 +217,18 @@ struct bt_iso_recv_info {
 	uint8_t flags;
 };
 
+/** @brief ISO Meta Data structure for transmitted ISO packets. */
+struct bt_iso_tx_info {
+	/** CIG reference point or BIG anchor point of a transmitted SDU, in microseconds. */
+	uint32_t ts;
```

Comment:
I do not know ISO well enough.  "ts" seems to indicate "time stamp" - is the time stamp of the packet equal to the reference point?

Reply:
I basically copied the text from the HCI_LE_Read_ISO_TX_Sync command response parameters. I admit to purposefully not adding more documentation here, as I don't understand all the nuances of the spec when it comes to timing. Vol 6, Part G, section 3.3 states that `The Host may determine the CIG reference point or BIG anchor point for the
last transmitted SDU from the TX_Time_Stamp and Time_Offset return param-
eters of the HCI_LE_Read_ISO_TX_Sync command`. So I think yes, this `ts` value equals the reference point for this particular SDU sequence number

Reply:
It's a good question. 

I would assume that (when https://github.com/zephyrproject-rtos/zephyr/pull/44117 is merged) an SDU is sent with timestamp=`T1` and sequence number=`S1`, and this function is called, then the timestamp of `S1` will be `T1` (assuming that it was sent on air).